webapp/packages/core-localization/src/locales/en.ts

```diff
@@ -14,7 +14,7 @@ export default [
   ['ui_processing_reload', 'Reload'],
   ['ui_processing_retry', 'Retry'],
   ['ui_processing_ok', 'Ok'],
-  ['ui_processing_create', 'Create'],
+  ['ui_processing_create', 'Finish'],
```

Comment:
in that case, we need to add a new key, `ui_processing_finish` to avoid confusing